cmp/grid/impl/GridPersistenceModel.js

```diff
@@ -95,7 +94,7 @@ export class GridPersistenceModel extends HoistModel {
         const {gridModel, state} = this;
         if (!state.columns) return;
 
-        const colState = this.cleanColumnState(state.columns);
+        const colState = gridModel.cleanColumnState(state.columns);
```

Comment:
just use setColumnState() here

Reply:
Probably to remove widths? IE:

// Remove the width from any non-resizable column - we don't want to track those widths as
// they are set programmatically (e.g. fixed / action columns), and saved state should not
// conflict with any code-level updates to their widths.

Reply:
10-4